docs/downstream-testing.md

```diff
@@ -10,7 +10,7 @@ In order to run this pipeline, your changes need to be available on the [main Ba
 
 ## Starting the tests
 
-After the new branch or Pull Request is created, visit the [Bazel (with downstream projects)](https://buildkite.com/bazel/bazel-with-downstream-projects-bazel/) pipeline. Then follow these steps:
+After the new branch or Pull Request is created, visit the [Bazel (with downstream projects)](https://buildkite.com/bazel/bazel-with-downstream-projects-bazel/) pipeline and select the Bazel project. Then follow these steps:
```

Comment:
Do you mean by selecting the Bazel organisation?

![image](https://user-images.githubusercontent.com/4171702/152760843-86ce44af-ac83-4717-b98f-8c5683c83a13.png)

Reply:
Here's what happened for me:
1) Follow the link (no "new build" option at top right)
2) Log in
3) Get redirected to https://buildkite.com/bazel
4) Select the Bazel project.
5) Proceed as normal

Not sure if that's normal, but it took me awhile (and the creation of a "new pipeline" by mistake) to realize I needed to select Bazel from https://buildkite.com/bazel after logging in. Happy for suggestions about how to clarify that better!

Reply:
This documentation is about the Bazel@HEAD + Downstream pipeline: https://buildkite.com/bazel/bazel-at-head-plus-downstream, which test Bazel built at HEAD with a set of downstream projects.

What are you actually trying to test here?